Berlin/BerOrOct59.xml

```diff
@@ -26,43 +30,250 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                      <seg type="script">eigheenth-century script.</seg>
+                      <seg type="ink">Black</seg>
+                        </handNote>
+
+                  <handNote xml:id="h7" script="Ethiopic" corresp="a5">
+                      <locus from="165r" to="166r"/>
+                      <seg type="ink">Black</seg>
+                        </handNote>
+
+                     </handDesc>
+
+
+                  <additions>
+                     <list>
+                        <item xml:id="a1">
+                           <locus from="143r" to="144v"/>
+                           <desc type="ProtectivePrayer"> Protective prayer with ʾasmāt. </desc>
+                           <q xml:lang="gez"> ቃለ፡ እግዚአብሔር፡ ዘመጽአ፡ ላዕለ፡ ዮርዳኖስ፡ ወተለውዎ፡ ማያት፡ ብዙኃን፡ ወኵሉ፡ ኃይል፡ ዘውስቴቶሙ፡ ወእለ፡ ሰሞሙ፡ ከመዝ፡ አልፋዝ፡ ሔድዛ፡
+                             ፆድዛ፡ ዑድዛ፡ በዝንቱ፡ አስማት፡
+                           </q>
+                        </item>
+
+                        <item xml:id="a2">
+                           <locus from="161v" to="162r"/>
+                           <desc type="CalendaricNote"> Calendaric note on the Apostles commemorative days in the months.
+                             Crudely written, probably a poor copy of <ref target="#a3"/>.</desc>
+                           <q xml:lang="gez"> ፬ለመስከረም፡ በርቴሎመዮስ፡ (sic) ፲ወ፪ለጥቅምት፡ ማቴዎስ፡ </q>
+                        </item>
+
+                        <item xml:id="a3">
+                           <locus from="163r" to="163v"/>
+                           <desc type="CalendaricNote"> Calendaric note on the Apostles commemorative days in the months. </desc>
+                           <q xml:lang="gez">  እም፡ ፬ለመስከረም፡ በርቴሎሜዎስ። አመ፡ ፲ወ፪ለጥቅምት፡ ማቴዎስ። አመ፡ ፳ወ፪ለጥቅምት፡ ሉቃስ።
+                           </q>
+                        </item>
+
+                        <item xml:id="a4">
+                           <locus from="164r" to="165r"/>
+                           <desc type="ProtectivePrayer"> Protective prayer with ʾasmāt. </desc>
+                           <q xml:lang="gez"> ነድ፡ ዘያንበለብል፡ ከማሁ፡ አንብልብል፡ ልሳኖ፡ ለገብረከ፡ እገሌ።
+                           </q>
+                        </item>
+
+                        <item xml:id="a5">
+                           <locus from="165r" to="166r"/>
+                           <desc type="ProtectivePrayer"> Protective prayer with ʾasmāt. </desc>
+                           <q xml:lang="gez"> ብርሃን፡ አጽፈ፡ ለወላዲ፡ ነበረ፡ ዲበ፡ ሰረገላ፡ ፼አእላፋት፡ ወትእልፊተ፡
+                           </q>
+                        </item>
+
+                        <item xml:id="a6">
+                           <locus target="#166v"/>
+                           <desc type="GuestText"> Letter of recommendation by the Franciscan Bartholomeo da Milano for two Ethiopian pilgrims,
+                             "Andrea and Peter Abassini", in the Holy Land to visit Rome </desc>
+                             <date when="1495" evidence="internal"/>
+                           <q xml:lang="it"> Italian
```

Comment:
This would mean that "Italian" is written on the page, which probably is not the case. If you just want to indicate the language without offering text, it would be `<q xml:lang="it"/>`